code/web/src/modules/survey/Survey.js

```diff
@@ -70,6 +75,24 @@ class Survey extends PureComponent {
 
         toggleSelection = (id) => {
           this.props.toggleSelection(id, this.props.survey.products)
+          this.setState(prevState=> ({
+            selectCount: prevState.selectCount + 1
+          }))
+        }
+
+        exitSurvey = () => {
+          confirm('Are you sure you want to exit the survey? Progress will be lost.')
+          this.props.clearSurvey()
+          this.props.history.push(crateRoutes.list.path)
+        }
+
+        retakeSurvey = () => {
+          console.log("THIS WORKS")
+          this.setState({
+            selectCount: 1
+          })
```

Comment:
Not totally clear on what this code does. Wondering if you could explain briefly. Happy to jump on a call if it's easier to show me than to type it all out.

Reply:
Actually I feel like I should be clearer: I see what this code does, I'm just unsure of why.

Reply:
It just makes it so the 'Next' button isn't disabled. A more thorough way to handle this would be to assign the 'selectCount' to the corresponding page, so that when a page is returned to, the number of items selected is the same. This would take an entire separate function.

Reply:
However, the purpose of the 'selectCount' is just to disable and enable selected items.

Reply:
Got it! Now the function to retake the survey will 'unselect' all of the products, so when a user returns to the beginning of the survey they have to select something to move on. Good catch!

Reply:
Awesome! Thank you, that solution is fire.